tests/test_listener.py

```diff
@@ -0,0 +1,39 @@
+from six.moves import mock
+
+from robotframework_reportportal.listener import listener
+
+ROBOT_SERVICE = 'robotframework_reportportal.service.ReportPortalService'
+
+
+class TestListener:
+
+    @mock.patch(ROBOT_SERVICE)
+    def test_critical_test_failure(self, mock_init, mock_variables,
+                                   suite_attributes, test_attributes):
+        mock_client = mock_init.return_value
```

Comment:
What is the purpose of this line?

Reply:
In my tests I perform end-to-end module testing. So my final point is `ReportPortalService` class from `client-python`. I mock it with a `@mock.patch` decorator above. But to get a mocked service object which was created with `self.rp = ReportPortalService( ... arguments ...)` I need to get `.return_value` attribute of my mock, otherwise it will be a class object.